conu/apidefs/metadata.py

```diff
@@ -0,0 +1,48 @@
+class Metadata:
+    """
+    Common metadata for container and image
+    """
+
+    def __init__(self, name=None, id=None, ports=None, labels=None, command=None,
+                 creation_timestamp=None, env_variables=None):
+        self.name = name
+        self.id = id
+        self.ports = ports
+        self.labels = labels
+        self.command = command
+        self.creation_timestamp = creation_timestamp
+        self.env_variables = env_variables
+
+
+class ContainerMetadata(Metadata):
+    """
+    Specific metadata for container
+    """
+
+    def __init__(self, name=None, id=None, ports=None, labels=None, command=None,
+                 creation_timestamp=None, env_variables=None,
+                 image=None, hostname=None, ipv4=None, ipv6=None, status=None):
+        super(ContainerMetadata, self).__init__(
+            name=name, id=id, ports=ports, labels=labels, command=command,
+            creation_timestamp=creation_timestamp, env_variables=env_variables)
+        self.hostname = hostname
+        self.image = image
+        self.ipv4 = ipv4
```

Comment:
we need to think a bit more about network addresses since one container can have more of them

Reply:
ipv4 and ipv6 can be lists, what do you think?

Reply:
can be some ports associated only to some IPs? will we represent this?

Reply:
@TomasTomecek ok I agree with name. @dhodovsk what about the dictionary
ipv4_adresses = {"10.0.0.1":8008, "10.0.02":8555} does it make sense?